flink-staging/flink-ml/src/main/scala/org/apache/flink/ml/common/ParameterMap.scala

```diff
@@ -112,7 +112,7 @@ object ParameterMap {
  *
  * @tparam T Type of parameter value associated to this parameter key
  */
-trait Parameter[T] {
+trait Parameter[+T] {
```

Comment:
This was initially needed to get the hack working, I will remove when I rework the ParameterMap